packages/@okta/vuepress-site/docs/guides/mfa/enroll-factor/sms/enrollfactor.md

```diff
@@ -0,0 +1,10 @@
+Using the ID for the User that you created earlier, add an SMS factor for that User:
+
+1. In Postman, open the **Factors (Okta API)** collection, and then the **Factor Lifecycle Operations** collection.
```

Comment:
no comma before and - only need a comma there if there is a separate action in the second clause.